src/screens/thank-you.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react'
+
+class ThankYouScreen extends Component {
+  render() {
+    return <h1 style={{textAlign:'center'}}>We've submitted your entry succesfully!</h1>
```

Comment:
Nope, the syntax highlighter doesn't have support for JSX in ES6 yet, and that unescaped ' breaks it.